pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -738,6 +738,10 @@ func (adc *attachDetachController) processVolumeAttachments() error {
 						err)
 					continue
 				}
+				klog.V(5).Infof("CSI Migration has been enabled for volume:%q, va.Name:%q, nodeName:%q",
```

Comment:
These functions are called very frequently, even after a volume has been successfully attached. Can we bump the log level to something very high like 10?

Reply:
Done